Formula/rocksdb.rb

```diff
@@ -24,8 +24,16 @@ class Rocksdb < Formula
   uses_from_macos "bzip2"
   uses_from_macos "zlib"
 
+  on_linux do
+    depends_on "gcc"
```

Comment:
Shouldn't this be a `:build` and `:test` dependency only?

Reply:
Not when compiling C++ code. Also has linkage with Homebrew GCC: https://github.com/Homebrew/homebrew-core/runs/5527422856?check_suite_focus=true#step:9:19